### DISCUSSION TITLE
Tag Naming

### DISCUSSION THREAD

**Discussion by devanteweary** [StuffAnThings/qbit_manage#469] [Q&A]
Hey everyone 👋

My question is this...
Is it possible to name my final tag something simple like "Seeding" or if not, something without a number like "Seeding_Unlimited"?

Currently the best I can do is have a tag called Seeding_1.Unlimited.
I'm trying to have as clean and easy to read tag as possible.

As far as I understand, qbitmanage combines "share_limits_tag:" with the name you set in "share_limits:" to create the name of the tag.

I have two categories under the "share_limits:" section that I want it to add the Seeding tag and force upload.

All I'm asking about is the actual name of the tag.
Am I missing something and it's easier to get the tag name you want?

Thank you!

**Comment by bakerboy448** [comment]
Yes — set `custom_tag` on the group in `share_limits:`. It replaces the auto-generated `{share_limits_tag}_{priority}.{group_name}` pattern entirely:
```yaml
share_limits:
  my_group:
    custom_tag: Seeding
    priority: 1
    ...
```
One constraint: the tag must be unique across all your share_limits groups — qbm uses it to determine which group a torrent belongs to, and rejects config at startup if two groups share a `custom_tag` (`modules/config.py`). See the `custom_tag` row in the [share_limits config table](https://github.com/StuffAnThings/qbit_manage/wiki/Config-Setup#sharelimits).
